rollup/internal/controller/watcher/chunk_proposer_test.go

```diff
@@ -141,6 +145,18 @@ func testChunkProposerLimits(t *testing.T) {
 			expectedChunksLen:          1,
 			expectedBlocksInFirstChunk: 1,
 		},
+		{
```

Comment:
can also add a multiple hard-fork test (although this would rarely happen).

Reply:
An additional test doesn't really provide additional coverage. Multiple hard-forks only matter for `BlocksUntilFork` which is propoerly tested [here](https://github.com/scroll-tech/scroll/pull/1174/files#diff-c614c84c333c0ed1b47977755aa8ff548f4263df17a46dc059af00c7645f4442R28)